_includes/icons/desktop-download.svg

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
let's run all the SVGs through an optimizer to save them bytes. Have you got SVGO on your machine? If not you can grab it here! https://github.com/svg/svgo-gui